.github/actions/next-release-cycle/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/grafana/agent/.github/actions/next-release-cycle
```

Comment:
Wouldn't it be better to put this in `/tools/next-release-cycle`? 
I would expect `.github` to really just contain YAML configs etc.